pkg/hoist/hoist_launchable.go

```diff
@@ -74,7 +74,17 @@ func (hl *Launchable) If() launch.Launchable {
 	return LaunchAdapter{Launchable: hl}
 }
 
+func (hl *Launchable) IsOneoff() bool {
```

Comment:
hmm that was just the hack so we don't change the runit directory layout for legacy customers. I sort of think of that as more of the uuid vs not-uuid split rather than oneoff vs long running